setup.cfg

```diff
@@ -8,8 +8,9 @@ filterwarnings =
 [flake8]
 max-line-length = 120
 max-complexity = 14
-inline-quotes = '
-multiline-quotes = """
+inline-quotes = single
+multiline-quotes = double
+avoid-escape = False
```

Comment:
Disable avoiding escaping same quotes in inline strings.
https://github.com/zheller/flake8-quotes#caveats

Reply:
I think this is the default, so not necessary. I generally try to avoid unnecessary settings.